spec/datetimeformat.html

```diff
@@ -123,15 +123,33 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
             1. Throw a *RangeError* exception.
           1. Let _timeZone_ be CanonicalizeTimeZoneName(_timeZone_).
         1. Set _dateTimeFormat_.[[TimeZone]] to _timeZone_.
-        1. Let _opt_ be a new Record.
+        1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
+        1. Let _formatOptions_ be a new Record.
+        1. Let _hcDefault_ be _dataLocaleData_.[[hourCycle]].
```

Comment:
This special use of hourCycle data feels to me like it's too early in the algorithm, can we instead move it to just before "Let _matcher_ be…" (i.e., the location in the current spec of "Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].")?

Reply:
yup that makes sense. I moved it down closer to matcher initialization :)